src/tutowidgets.wiki

```diff
@@ -197,7 +201,11 @@ let mywidget s1 s2 =
   div ~a:[a_class ["mywidget"]] [button; content]
 
 let _ =
-  Ex_app.register_service ~path:[] ~get_params:Eliom_parameter.unit
+  Ex_app.register
+    (Eliom_service.create
+       ~id:(Eliom_service.Path [])
+       ~meth:(Eliom_service.Get Eliom_parameter.unit)
+       ())
```

Comment:
Why not `Ex_app.create`, which creates a service and registers it with the same function? (Same issue repeated later.)